app/src/main/res/xml/preferences.xml

```diff
@@ -46,6 +46,12 @@
 	/>
 
 	<CheckBoxPreference
+		android:key="showBreakdown"
+		android:title="@string/pref_breakdown"
+		android:summary="@string/pref_breakdown_summary"
+	/>
```

Comment:
I missed this in the last review, but this and the other new preferences should have a `defaultValue` which is false for this, and "score by word length" for the other one.